sympy/physics/hydrogen.py

```diff
@@ -177,15 +183,18 @@ def E_nl_dirac(n, l, spin_up=True, Z=1, c=Float("137.035999037")):
     The energy is calculated from the Dirac equation. The rest mass energy is
     *not* included.
 
-    n, l
+    Parameters
+    ==========
+
+    n, l :
         quantum numbers 'n' and 'l'
-    spin_up
+    spin_up :
         True if the electron spin is up (default), otherwise down
-    Z
+    Z :
         atomic number (1 for Hydrogen, 2 for Helium, ...)
-    c
+    c :
         speed of light in atomic units. Default value is 137.035999037,
-        taken from: http://arxiv.org/abs/1012.3627
+        taken from [1].
```

Comment:
Have you checked, if this is rendered correctly?

Reply:
Its not getting rendered correctly. I replaced it by its original link.

Reply:
Resolved.